server.js

```diff
@@ -71,10 +71,28 @@ async function renderAndCache(req, res, pagePath, serverData) {
       appMiningMonths,
       appMiningApps,
       params: req.params
+    };
+
+    // console.log('jwt', req.universalCookies.get('jwt'));
+
+    const token = req.query.authResponse;
+    // console.log('token', {token});
+
+    if (token) {
+      const authUrl = `${apiServer}/api/authenticate?authToken=${token}`;
+      const authResp = await request({
+        method: 'POST',
+        uri: authUrl,
+        json: true
+      });
+      if (authResp.success) {
```

Comment:
Seems like we also want to set the cookie here, too, and then remove the cookie-setting stuff on the front-end.